airflow/cli/commands/info_command.py

```diff
@@ -0,0 +1,427 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Config sub-commands"""
+import getpass
+import locale
+import logging
+import os
+import platform
+import subprocess
+import sys
+import textwrap
+from typing import Optional
+from urllib.parse import urlsplit, urlunsplit
+
+import requests
+import tenacity
+from typing_extensions import Protocol
```

Comment:
This should be `from airflow.typing_compat import Protocol`  -- on Py 3.8+ it will use the built in one then.

Reply:
https://github.com/python/typing/blob/master/typing_extensions/src_py3/typing_extensions.py#L1116-L1119
I can not see the difference. We can't delete typing_extension anyway. This is still needed in many places.